Kubernetes-CLI.adoc

```diff
@@ -0,0 +1,63 @@
+= Kubernetes CLI
+
+The Infinispan CLI currently has two modes: command-line and interactive. 
+To improve the user experience when using the Infinispan Operator we can enhance the CLI to add Kubernetes capabilities.
+
+== Native
+
+Using Quarkus we can build the CLI as a native executable for the three most common platforms: Linux x86_64, OS X, Windows.
+
+== Kubernetes mode
+
+Installing the executable as `kubectl-infinispan` allows invocation as a `kubectl/oc` plugin, e.g.:
+
+`kubectl infinispan ...`
+
+Using the Fabric8 Kubernetes Client we can automatically detect the existence of a Kubernetes configuration (default to `~/.kube/config`) and automatically connect to the Kubernetes cluster.
+
+=== Kubernetes commands
+
+The Infinispan Kubernetes CLI plugin defaults to using the `datagrid` namespace and the `infinispan` service name. These can be overridden in each command using the `--namespace` parameter and the name argument.
```

Comment:
Default namespace should default to current context (can be retrieved eg by `oc project` or found in config). If the plugin will default to different namespace it will create a confusion. `kubectl/oc` would operate on top of different namespaces depending on whether user is using core `oc/kubectl` or operating with the plugin.

Reply:
I'll amend that.